test/passes/O.bin.txt

```diff
@@ -51,9 +49,10 @@
   )
   (loop $label$3
    (if
-    (i64.ne
-     (local.get $0)
-     (i64.const 0)
+    (i32.eqz
+     (i64.eqz
```

Comment:
It seems `i32.eqz i64.eqz` could be eliminate at all for CFG operands

Reply:
What do you mean by "CFG" here?

Reply:
I meant arguments for control operations like `if (i64)` but I forgot that `if` should be accept only `i32` input argument, so series of `eqz` may be better than `i64.ne` + `i64.const 0`